libnfc/buses/uart.c

```diff
@@ -367,6 +384,26 @@ uart_receive(serial_port sp, uint8_t *pbtRx, const size_t szRx, void *abort_p, i
   return NFC_SUCCESS;
 }
 
+/**
+ * @brief Send \a pbtTx content to UART one byte at a time
+ *
+ * @return 0 on success, otherwise a driver error is returned
+ */
+int
+uart_send_single(serial_port sp, const uint8_t *pbtTx, const size_t szTx, int timeout)
+{
+  (void) timeout;
+  int error = 0;
+  for (int i = 0; i < szTx; i++)
+  {
+    error |= uart_send(sp, pbtTx+i, 1, timeout);
```

Comment:
Here, the error handling shallow different kind of error (e.g. `NFC_EOPABORTED`, `NFC_ETIMEOUT`, etc.).

Next, this way you will continue to send even an error occurred.

Reply:
Good call! Fixed in 4c50adc.